### PR TITLE
Cu 86b0ypuc6 save posture data

### PR DESCRIPTION
**Pull request by whamester**

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b0yptay Create a Challenge](https://app.clickup.com/t/86b0yptay)
Task linked: [CU-86b0ypuc6 Save Posture Data](https://app.clickup.com/t/86b0ypuc6)

File: src/app/(tabs)/index.tsx [hunk 1]
```diff
@@ -25,6 +25,10 @@ const background = {
 const HomePage = () => {
   const isSetupComplete = useUser((state) => state.isSetupComplete);
   const userName = useUser((state) => state.user.name);
+  const userLevel = useUser((state) => state.user.level);
+  const userHP = useUser((state) => state.user.hp);
+  const userXP = useUser((state) => state.user.xp);
+  const userSteak = useUser((state) => state.user.daily_streak_counter);
```
Comment (Victorportus) [inline src/app/(tabs)/index.tsx]: typo: Steak instead of streak

File: src/app/(tabs)/index.tsx [hunk 2]
```diff
@@ -122,7 +125,7 @@ const HomePage = () => {
               justifyContent="start"
             >
               <Image source={require("../../../assets/img/avatar.png")} />
-              <Text>5 Day Streak</Text>
+              <Text>{userSteak} Day Streak</Text>
```
Comment (Victorportus) [inline src/app/(tabs)/index.tsx]: typo: Steak instead of streak

**Review by Victorportus — APPROVED** [review]
Leaved a typo comment, It runs on my side and have the log of the task.

File: src/app/(tabs)/index.tsx (hunk 1 above)
Comment (ReinhardtBotha) [reply src/app/(tabs)/index.tsx]: user 🥩

Comment (whamester) [reply src/app/(tabs)/index.tsx]: fixed 🥩

**Pull request merged**